### PR TITLE
React-Capstone

### PR DESCRIPTION
**Pull request by muneeb3123**
This PR includes:

- one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.
- ge for the item details; in the example, the detail page for Czech Republic cities with number of views.

### CODE REVIEW

**Comment by muneeb3123** [comment]
If you want changes kindly connect me on Zoom or Slack. i will available to make changes. Thanks for review

File: README.md [hunk 1]
```diff
@@ -1,70 +1,143 @@
...
+- React
+- Linters
+- CSS
+<!-- Features -->
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## 🚀 Live Demo <a name="protfolio-project"></a>
+ 
+https://tourmaline-nougat-967856.netlify.app
```
Comment (brytebee) [inline README.md]: This link opens to an empty page that only prints the string "hy".

Fix the deployment.

image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,143 @@
...
-### `npm run build`
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+ 📼 Project Walkthrough <a name="project-walk"></a>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+https://www.loom.com/share/756c3a2ea73843c5ac9d11d32e89a8dd
```
Comment (brytebee) [inline README.md]: Great video presentation. 👏👏👏

However, a few things need to be added.
- You are required to highlight some interesting piece of code or something you built that you are very proud of.

image

File: src/components/Home/Home.js
Comment (brytebee) [inline src/components/Home/Home.js:1]: Great design and meets the requirements. 👏👏👏

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @muneeb3123 👋, 

##

### Highlights💡
- Good dynamic routing with useNavigate hook.✔️ 
- Great state management with useState hook.✔️
- Great use of redux to manage the global state.✔️
- Great use of promises and REST API.✔️
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

----



----

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @muneeb3123 👋, 

##

### Highlights💡
- Good dynamic routing with useNavigate hook.✔️ 
- Great state management with useState hook.✔️
- Great use of redux to manage the global state.✔️
- Great use of promises and REST API.✔️
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

----



----

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Home/Home.js
Comment (muneeb3123) [reply src/components/Home/Home.js:1]: Thanks

File: README.md (hunk 1 above)
Comment (muneeb3123) [reply README.md]: FIxed

**Review by shella12 — APPROVED** [review]
Hi @muneeb3123, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Live demo link has been fixed ✔️ 
- Video presentation highlights the code 👍 
- Great design implementation 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**